packages/schemas/src/foundations/jsonb-types/account-centers.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from 'zod';
+
+export enum AccountCenterControlValue {
+  Off = 'Off',
+  ReadOnly = 'ReadOnly',
+  Edit = 'Edit',
+}
+
+// Control list of each field in the account center (profile API)
+// all fields are optional, if not set, the default value is `Off`
+// this can make the alteration of the field control easier
```

Comment:
nit: Moving these comments to the TypeScript type definition as JSDoc comments is better. They will appear in IntelliSense, making them visible to everyone using it.